### DISCUSSION TITLE
Add "Brightness Only" option

### DISCUSSION THREAD

**Discussion by zyberwoof** [basnijholt/adaptive-lighting#305] [Ideas]
Use the 3 graphs at the [bottom of here](https://github.com/basnijholt/adaptive-lighting) for reference.  You can see that the intention is to do most of the changes by adjusting the color temperature.  The brightness graph starts at zero and quickly plateaus at 100%.  Once the brightness tops out at 100%, the rest of the changes are done more gradually via the temperature.

When using lights that adjust by brightness only, the effects are quite jarring.  The lights quickly end up at 100% in the early morning and remain there for most of the day.  

I'd like to suggest a "brightness_only" option.  This option would make the brightness graph more gradual, and the brightness would be changing throughout the day.  With this, the color temperature would likely remain static.  

An alternative would be an option like "gradual_brightness_adjustment" or "elongate_brightness_graph".  With this you can assume the temperature changes like normal.  But, like above, the brightness would gradually increase and decrease throughout the day with the position of the sun.

**Comment by broyuken** [comment]
Adjust the sunrise and sunset offsets, I have some brightness only ones that I do 11:59 sunrise and 12:01 sunset.

**Comment by daFreeMan** [comment]
@broyuken could you please expand on how/where to make the sunrise/sunset offsets. 

I very much like the color temperature changing, but find that, especially now that it's dark outside when I get up, the lights are _far_ too bright first thing in the morning and would like to have them brighten more slowly and dim more quickly in the evening for general lighting purposes.

Of course, if I need bright task lighting, I have no objection to manually setting brightness & color temps.

**Comment by broyuken** [comment]
Go into your HA integrations, pick the AL integration for the lights you want to adjust, click configure and it’s one of the boxes in there.

**Comment by daFreeMan** [comment]
are you talking about this:
![image](https://user-images.githubusercontent.com/11889733/192830585-c4cf484a-e4ed-4761-8189-5de55166cb38.png)

**Comment by broyuken** [comment]
Yes the sunrise_offset and sunset_offset are the ones
